core/src/components/item-options/item-options.scss

```diff
@@ -33,7 +33,7 @@ ion-item-options {
 
   height: 100%;
 
-  font-size: 14px;
+  font-size: dynamic-font(14px);
```

Comment:
This shouldn't actually be used anywhere since it is overridden by `item-option` styles, but I didn't want to remove it because I don't know if anyone is using an `ion-item-options` element without `ion-item-option` children and relying on this `font-size` to exist.

Reply:
Created FW-5020 to investigate removing this.